contracts/smart-account/interfaces/modules/IERC7484SecurityPolicyPlugin.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {ISecurityPolicyPlugin} from "contracts/smart-account/interfaces/modules/ISecurityPolicyManagerPlugin.sol";
+import {IERC7484SecurityPolicyPluginEventsErrors} from "contracts/smart-account/interfaces/modules/IERC7484SecurityPolicyPluginEventsErrors.sol";
+
+/**
+ * @title ERC7484 Security Policy Plugin
+ * @author @ankurdubey521
+ * @notice Second Order Plugin to the Security Policy Manager Plugin, enforces checks as defined by ERC7484
+ */
+// https://eips.ethereum.org/EIPS/eip-7484
+interface IERC7484SecurityPolicyPlugin is
+    ISecurityPolicyPlugin,
+    IERC7484SecurityPolicyPluginEventsErrors
```

Comment:
do you need this contract really?

IERC7484SecurityPolicyPlugin can itself contain struct, errors definition, events definition, and other function interfaces. We have been following this for other modules / plugins

Reply:
Yeah, I really don't like having a separate contract as well.

The reason the events and errors are in a separate interface is because I need to inherit these in the foundry tests, so that I have access to all event and error definitions.
If I keep everything in one interface, inheriting it in the tests would require implementing all the functions in the test as well, which is not ideal.

Reply:
I see. Can you not write interfaces just for tests though? compromising in tests is better than compromising on contracts

Reply:
That would lead to duplicating events which defeats the purpose of having interfaces here